.github/workflows/nodejs.yml

```diff
@@ -52,6 +51,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
       - run: yarn install
+      - run: yarn build
```

Comment:
You're right for the time being, though it's strange to run it independently IMO, and we'll need to do this anyways to eventually test against our built files instead of src (if that's still something we want to do).

Reply:
Meh, it's not like `pre-build` is a private command, it's broken out for convenience. And it's not like our CI build scripts follow any conventions, they just do what they need to test what we want.

Reply:
Fair enough, will change that